EESSI-install-software.sh

```diff
@@ -285,9 +285,10 @@ ec=$?
 if [ ${ec} -eq 0 ]; then
     module load ${eessi_extend_module}
 else
-    echo "Did not find ${eessi_extend_module} module; setting EASYBUILD_INSTALLPATH and EASYBUILD_EXPERIMENTAL manually"
+    echo "Did not find ${eessi_extend_module} module; installing it with latest EasyBuild"
+    # EESSI-extend is the thing that sets EASYBUILD_INSTALLPATH so we have a chicken/egg situation to resolve
     export EASYBUILD_INSTALLPATH=${EESSI_PREFIX}/software/${EESSI_OS_TYPE}/${EESSI_SOFTWARE_SUBDIR_OVERRIDE}
-    export EASYBUILD_EXPERIMENTAL=1
+    source load_eessi_extend_module.sh ${EESSI_VERSION}
```

Comment:
Because the new script checks if the module exists, the whole block starting at line 282 and ending at line 292 could be replaced with the `source load_eessi_extend...`?

Reply:
Almost, issue is that the script relies on `$EASYBUILD_INSTALLPATH` ... so setting this plus sourcing the script could be sufficient.